doc/source/examples/unresolved-cfd-dem/gas-solid-spouted-cylinder-bed/gas-solid-spouted-cylinder-bed.rst

```diff
@@ -0,0 +1,300 @@
+==================================
+
+This example is an extension of the `Gas-Solid Spouted Bed <../gas-solid-spouted-bed/gas-solid-spouted-bed.html>`_ for a cylindrical geometry. 
+
+----------------------------------
+Features
+----------------------------------
+
+- Solvers: ``lethe-particles`` and ``lethe-fluid-particles``
+- Three-dimensional problem
+- Simulates a solid-gas cylinder-shaped spouted bed
+
+---------------------------
+Files Used in This Example
+---------------------------
+
+Both files mentioned below are located in the example's folder (``examples/unresolved-cfd-dem/gas-solid-spouted-cylinder-bed``).
+
+- Parameter file for CFD-DEM simulation of the spouted bed: ``gas-solid-spouted-cylinder-bed.prm``
+- Parameter file for particle generation and packing: ``dem-packing-in-spouted-cylinder-bed.prm``
+
+-----------------------
+Description of the Case
+-----------------------
+
+This example simulates the spouting of spherical particles in air in a cylinder. As noted in the example of `Gas-Solid Spouted Bed <../gas-solid-spouted-bed/gas-solid-spouted-beml>`_, we use ``lethe-particles`` to fill the bed with particles, and ``lethe-fluid-particles`` as a solver of CFD-DEM.
+
+-------------------
+DEM Parameter File
+-------------------
+
+Here, we will focus only on the parts that have been modified. It is strongly recommended to visit `DEM parameters <../../../parameters/dem/dem.html>`_ for a detailed description on the concepts and physical meanings of the DEM parameters.
+
+Mesh
+~~~~~
+
+In this example, we are simulating a cylinder shaped spouted bed. We introduce the flow through a cylinder of smaller radius channel that  constitutes the inlet of the bed. A schematic image is shown below;
+
+.. image:: images/geometry.png
+    :alt: The geometry and boundary conditions
+    :align: center
+    :name: geometry
+    :height: 10cm 
+
+The geometry of the bed was created using `Pointiwise <../../../tools/pointwise/pointowise.html>`_, and the overview of created mesh is also shown below;
+
+.. image:: images/mesh.png
+    :alt: The geometry and boundary conditions
+    :align: center
+    :name: mesh_ver
+    :height: 10cm
+
+In Unresolved-CFD-DEM, Cells need to have a enough volume to contain particles in it. Basically the size of cells should be least three time larger than the diameter of the particles so that the void fraction would not be discontinuous `[1] <https://doi.org/10.1002/cjce.23686>`_. So this can be expressed as follows.
+
+.. math:: 
+  \dfrac{d_p}{\Delta x} \leq 3
+
+where :math:`d_p` is the particle diameter, and :math:`\Delta x` is the characteristic size of the cell. In this example, we use particles with diameters of 5 mm which means that we need at least 15 mm for cell size. Also, we need to apply a relatively small mesh to the short channel below. Thus, we set the size of the central cylinder to 15 mm, and made other grid coarser. We show the parameter related to the mesh below.
```

Comment:
I had written a comment regarding this previously. What you changed here is also not true. The averaging volume does not need to be related to the cells at all...